pandas/tests/series/methods/test_describe.py

```diff
@@ -163,6 +166,12 @@ def test_numeric_result_dtype(self, any_numeric_dtype):
             dtype = "complex128" if is_complex_dtype(any_numeric_dtype) else None
 
         ser = Series([0, 1], dtype=any_numeric_dtype)
+        if dtype == "complex128" and is_numpy_dev:
+            with pytest.raises(
+                TypeError, match=r"^a must be an array of real numbers$"
```

Comment:
Do you know why this isn't allowed anymore?

Reply:
I think it should always have raised - complex numbers aren't ordered, I don't see how it used to work

Reply:
indeed https://github.com/numpy/numpy/issues/22652